### DISCUSSION TITLE
How get relation manager resource (grand-child) in other relation manager resource

### DISCUSSION THREAD

**Discussion by jigujigu** [filamentphp/filament#6344] [Help] [answered]
Hi ALl,

I Have filament RelationManager in my some filament resource  .
My Question is : is it possible  to have another relationmanager (as grand-child) in relation manager ?

Something like relationship between parent with child, then child with grand-child.

Thank you.

**Comment by danharrin** [comment, marked as answer]
Not a relation manager, but you could nest a relationship() Repeater inside

**Reply by jigujigu** [reply]
Ok. Thanks
